examples/01_datasets/05_plot_nimads.py

```diff
@@ -0,0 +1,60 @@
+"""
+
+.. _nimads_object:
+
+========================
+Using NIMADS with NiMARE
+========================
+
+How to use the NeuroImaging Meta-Analysis Data Structure
+`(NIMADS) <https://neurostuff.github.io/NIMADS/>`_ with NiMARE.
+"""
+from requests import request
+
+from nimare.io import convert_nimads_to_dataset
+from nimare.nimads import Studyset
+
+###############################################################################
+# Download Data from NeuroStore
+# -----------------------------------------------------------------------------
+
+
+def download_file(url):
```

Comment:
Minor comment: I'm not sure this function is that useful since you only use it twice (below). 

Maybe you could make this more useful and use it throughout

Reply:
yeah, my threshold for wanting to write a function was pretty low, it's possible other examples could be downloaded down the line using this function.